internal/scorecard/alpha/formatting.go

```diff
@@ -30,13 +31,13 @@ func getTestResults(client kubernetes.Interface, tests []ScorecardTest) (output
 		p := tests[i].TestPod
 		logBytes, err := getPodLog(client, p)
 		if err != nil {
-			fmt.Printf("error getting pod log %s\n", err.Error())
+			log.Errorf("Error getting pod log %s\n", err.Error())
```

Comment:
Rather than logging an error, we should convert this error to a `ScorecardTestResult` and append it to the list of results.

Reply:
fixed.